### PR TITLE
refactor: migrate to vite and vitest

### LINKED ISSUE #96: Migration to vite
Following #95, try migrating the project to `vite`. The current `main` branch had changes in structure so we don't like to directly merge the PR.

### LINKED ISSUE #97: Remove fbt
React fbt is not well maintained since 2021.

We'd like to remove this.

### PR DESCRIPTION
**Pull request by DevYuns**
resolves #96, resolves #97

## Description
Remove `fbt` stuff then Migration to [vite](https://vitejs-kr.github.io/) and [vitest](https://vitest.dev/).

### CODE REVIEW

**Comment by DevYuns** [comment]
@hyochan I finished the migration job.

**Comment by DevYuns** [comment]
@hyochan  👀

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Install modules
-        run: yarn
+        run: YARN_ENABLE_IMMUTABLE_INSTALLS=false yarn
```
Comment (hyochan) [inline .github/workflows/ci.yml:17]: It'd be good if you can write a comment on this

File: README.md [hunk 2]
```diff
@@ -70,8 +69,7 @@ app/
 ├─ STRINGS.js
 ├─ tsconfig.json
 ├─ typings.d.ts
-├─ webpack.config.js
-└─ webpack.config.prod.js
+├─ vite.config.ts
```
Comment (hyochan) [inline README.md:72]: Prefer `└─` at the end please.

**Review by hyochan — APPROVED** [review]
Looks great overall. Please write a better PR message next time tough~

**Review by hyochan — APPROVED** [review]

**Pull request merged**